image_proc/src/crop_decimate.cpp

```diff
@@ -356,6 +360,53 @@ void CropDecimateNode::imageCb(
   pub_.publish(out_image, out_info);
 }
 
+rcl_interfaces::msg::SetParametersResult CropDecimateNode::paramCb(
+  const std::vector<rclcpp::Parameter>& parameters)
+{
+  rcl_interfaces::msg::SetParametersResult result;
+  result.successful = true;
+  for (const auto & param : parameters) {
+    if (param.get_name() == "offset_x") {
```

Comment:
Should other parameters be handled as well?

Reply:
you can also add `decimation_x_`, `decimation_y_` and `target_frame_id_`